tests/support/server.tcl

```diff
@@ -447,6 +447,7 @@ proc start_server {options {code undefined}} {
     set keep_persistence false
     set config_lines {}
     set start_other_server 0
+    set old_singledb $::singledb
```

Comment:
is this a backup of the initial value set on start up?

Reply:
It's a backup of the previous value, the value it has when entering the `start_server` block. These things can be nested.

```
tags {singledb} {
    start_server {tags {singledb}} {
        test {bla bla bla} {
            ...
        } {} {singledb}
    }
}
```